andela/ah-infinity-stones-frontend
# #161966868 Setup Sass Preprocessor

#### What does this PR do?
- Integrates `.sass` and `.scss` style compiler
- Allows app to import `.scss`
- Allows app to build or compile into `.css` 

#### Description of Task to be completed?
> Install node-sass
> Write a sass or scss styles
> Run to compile and navigate to any UI pages using css style in the browser

#### How should this be manually tested?
`$ git clone -b ch-sass-setup-161966868 https://github.com/andela/ah-infinity-stones-frontend`
`$ cd ah-infinity-stones-frontend`
> Install node modules specified in `package.json`
`$ npm install`
`$ npm start`
> Now you are ready to edit and see own changes
> Navigate to `./src/css/` directory and edit the `main.scss`
> Import this stylesheet in `index.js` below all styles with import './css/main.scss';` 
> Now you should see the page relaod with new style
> `$ npm run build-css` to build css
> Now you should see the compiled `.css` with the name used above


#### What are the relevant pivotal tracker stories?
[#161966868](https://www.pivotaltracker.com/story/show/161966868)


#### Screenshots
<img width="1243" alt="screen shot 2019-01-11 at 12 11 23" src="https://user-images.githubusercontent.com/39657549/51024245-1308c700-159a-11e9-862c-2b5548063352.png">

<img width="1243" alt="screen shot 2019-01-11 at 12 14 16" src="https://user-images.githubusercontent.com/39657549/51024368-71ce4080-159a-11e9-9206-1e543cbc1ee2.png">

<img width="1272" alt="screen shot 2019-01-09 at 13 28 30" src="https://user-images.githubusercontent.com/39657549/50893727-97c8d900-1412-11e9-92ec-fd272edcdf75.png">

<img width="1279" alt="screen shot 2019-01-11 at 10 10 21" src="https://user-images.githubusercontent.com/39657549/51018584-2c554780-1589-11e9-9d5e-e67c49e66d23.png">
